ext/sqlite3/tests/gh9032.phpt

```diff
@@ -0,0 +1,28 @@
+--TEST--
+SQLite3 authorizer crashes on NULL values
+--SKIPIF--
+<?php
+if (!extension_loaded("sqlite3")) die("skip sqlite3 extension not available");
+?>
+--INI--
+open_basedir=.
+--FILE--
+<?php
+$db = new SQLite3(":memory:");
+$db->enableExceptions(true);
+
+$db->exec('attach database \':memory:\' AS "db1"');
+var_dump($db->exec('create table db1.r (id int)'));
+
+try {
+    $st = $db->prepare('attach database :a AS "db2"');
+    $st->bindValue("a", ":memory:");
```

Comment:
I discovered this issue by using a database abstraction layer which bind strings as params. Is there any easy way to support the `ATTACH DATABASE` with value passed by param together with `open_basedir`?

Reply:
👍 I see, it is also documented in https://www.php.net/manual/en/sqlite3.setauthorizer.php and it seems the only supported stage by Sqlite http://sqlite.org/c3ref/set_authorizer.html

Reply:
Ah, right, for ext/sqlite3 you could overwrite the authorizer callback; that is not (yet) supported for ext/pdo_sqlite, though.